locale/en/blog/release/v0.10.0.md

```diff
@@ -317,7 +317,7 @@ versions of Node.  This is a remarkably effective way to do node-core
 development.  Future developments will continue to be iterated in
 userland modules.
 
-## Growing Up <a name="enterprise"></a>
+## <!--enterprise-->Growing Up
```

Comment:
Previously this resulted in `#growing-up-a-name-enterprise-a`

Reply:
Not sure if I should just add `<a id="enterprise></a>` above, for backwards compatibility.